frontend/webapp/views/main/templates/hijack.htm

```diff
@@ -539,22 +589,61 @@ <h5 class="modal-title" id="deleteHijacksModalLabel">Delete Hijack</h5>
+                        let count_query = [];
+                        let data_query = [];
+
+                        var sort;
+                        if(data.order[0].dir == 'desc'){
+                            sort = 'desc_nulls_first';
+                        }else{
+                            sort = 'asc';
+                        }
                         datatable_parameters['offset'] = data.start;
                         datatable_parameters['limit'] = data.length;
-                        datatable_parameters['order'] =  columns[data.order[0].column].data + "." + data.order[0].dir;
-                        $.ajax({
-                            type: "POST",
-                            url: "{{ url_for('proxy_api') }}",
-                            dataType: "json",
-                            data: { "action": "view_bgpupdates", "parameters" : JSON.stringify(datatable_parameters) },
-                            success: function(data, status, xhr){
-                                callback({
-                                    recordsTotal: data.total,
-                                    recordsFiltered: data.total,
-                                    data: format_bgp_updates_datatable(data.results)
-                                });
-                            }
-                        });
+
+                        count_query.push(" count_data: ");
+                        count_query.push(hasura['extra']['table_name']);
+                        count_query.push("_aggregate");
+
+                        if(datatable_parameters['and'] != null){
+                            count_query.push("(");
+                            count_query.push(datatable_parameters['and']);
+                            count_query.push(") { aggregate { count } } ");
+                        }else{
+                            count_query.push(" { aggregate { count } } ");
+                        }
+
+                        data_query.push("view_data: ");
+                        data_query.push(hasura['extra']['table_name']);
+                        data_query.push("(limit: ");
+                        data_query.push(datatable_parameters['limit']);
+                        data_query.push(", offset: ");
+                        data_query.push(datatable_parameters['offset']);
+                        data_query.push(", order_by: {");
+                        data_query.push(columns[data.order[0].column].data);
+                        data_query.push(": ");
+                        data_query.push(sort);
+                        data_query.push(" } ");
+
+                        if(datatable_parameters['and'] != null){
+                            data_query.push(datatable_parameters['and']);
+                        }
+                        data_query.push(") { ");
+                        data_query.push(hasura_fetch.join(' '));
+                        data_query.push(" }");
+
+                        count_open_details = 0;
```

Comment:
what does this do?

Reply:
Counts expanded BGP updates.

Reply:
Is it reinitiated whenever one moves to the next page or refreshes the page? Otherwise it will simply go up all the time as the user navigates from page to page (table to table I mean)